core/src/main/java/com/sequenceiq/cloudbreak/repository/ImageCatalogRepository.java

```diff
@@ -51,4 +51,11 @@
 
     Optional<ImageCatalog> findByResourceCrnAndArchivedFalseAndImageCatalogUrlIsNotNull(String resourceCrn);
 
+    @Query("SELECT i FROM ImageCatalog i "
+            + "LEFT JOIN FETCH i.workspace w "
+            + "LEFT JOIN FETCH w.tenant t "
+            + "WHERE i.id IN :ids "
+            + "AND i.archived = false")
```

Comment:
I am not sure if the `archived = false` will change some logic or not, can you check?